front-end/src/Employee/Registration/employee-registration-types.tsx

```diff
@@ -0,0 +1,33 @@
+export interface EmployeeRegistrationFormProps {
+  firstName: string;
+  middleName: string;
+  lastName: string;
+  idNumber: string;
+  dob: string;
+  age: string;
+  telephone: string;
+  email: string;
+  gender: string;
+  kraPin: string;
+  nssf: string;
+  nhif: string;
+  pfNumber: string;
+  salutation: string;
+}
+
+export const formValues: EmployeeRegistrationFormProps = {
+  firstName: '',
+  middleName: '',
+  lastName: '',
+  idNumber: '',
+  dob: '',
+  age: '',
```

Comment:
We should not be capturing the age since this is a calculated field and changes, having Date of Birth is enough to have age information